SYCL/Basic/image/image_sample.cpp

```diff
@@ -1,7 +1,8 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
-// UNSUPPORTED: cuda || hip
+// Temporarily disable test on Windows due to regressions in GPU driver.
+// UNSUPPORTED: cuda || hip, windows
```

Comment:
Why it is unsupported?
I guess it should be XFAIL:level_zero

Reply:
XFAIL is a poor test disabling practice. When the test is fixed, it'll fire up in every test run.

Reply:
this is intended. Otherwise we may miss fix/unstable tets behavior.

Reply:
+ 1 for XFAIL. XFAIL shows a problem in implementation which must be fixed, UNSUPPORTED means the test is not meant to be run at all on unsupported config.

Reply:
I think the comment clearly explains the motivation to disable test on Windows. I agree with Alex that using UNSUPPORTED to disable failing tests in llvm-test-suite is less disruptive for CI system than XFAIL.
As of today, it's not possible to make some commits to intel/llvm and do not break CI checks if we use XFAIL mark.